topics/chrome/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: google-chrome
+related: firefox,safari,edge,opera,chromium
+created_by: Google
+display_name: Chrome
+github_url: https://github.com/googlechrome
+logo: chrome.png
+released: September 02, 2008
+short_description: Chrome is a web browser from the tech company Google.
+topic: chrome
+url: https://www.google.com/chrome/
+---
+Chrome is a web browser from the tech company Google. The most popular web browser worldwide as of mid-2017. Availble in desktop, mobile and tablet flavours.
```

Comment:
What about combining the first two:

> Chrome is the most popular web browser worldwide as of mid-2017, made by the tech company Google.

Typo: `Availble`

What about rephrasing the last part:

> It is available on the desktop, phones, and tablets.

Could you add any more to this description, maybe something about how Chrome differs from other popular browsers? You could also note the operating systems it's available in.